protocol-units/maptos-rest/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+use anyhow::Error;
+use aptos_api::Context;
+use poem::listener::TcpListener;
+use poem::{
+	get, handler,
+	middleware::{AddData, Tracing},
+	web::{Data, Path},
+	EndpointExt, IntoResponse, Response, Route, Server,
+};
+use std::env;
+use std::sync::Arc;
+use tracing::info;
+
+#[derive(Debug)]
+pub struct MaptosRest {
```

Comment:
Here also, let's not promote "maptos" into an official naming prefix for actual item names. This quirkiness is only needed to disambiguate in flat namespaces like crate names (when published in a registry) and environment variables. Just use `Server` so it's imported as `use maptos_rest::Server`?

Reply:
Also, perhaps `movement-rest`?

Reply:
The name was suggested by Liam when I was asking about how to name the service. But I agree the quirkiness is not needed here!